app/controllers/admin/halls_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Admin
+  class HallsController < Admin::ApplicationController
+    # Overwrite any of the RESTful controller actions to implement custom behavior
```

Comment:
I know that these comments are very useful at the beginning of the journey with Administrate. In the future, it would be nice to get rid of them or leave them only in one Administrate's controller. Currently you have a lot of duplicated comments in many files. The same story with Administrate's dashboards. The decision is up to you.